samples/Nancy.Demo.Hosting.Kestrel/AppConfiguration.cs

```diff
@@ -0,0 +1,29 @@
+namespace Nancy.Demo.Hosting.Kestrel
+{
+    public class LogLevel
+    {
+        public string Default { get; set; }
+        public string System { get; set; }
+        public string Microsoft { get; set; }
+    }
+
+    public class Logging
+    {
+        public bool IncludeScopes { get; set; }
+        public LogLevel LogLevel { get; set; }
+    }
+
+    public class Smtp
+    {
+        public string Server { get; set; }
+        public string User { get; set; }
+        public string Pass { get; set; }
+        public string Port { get; set; }
+    }
+
+    public class AppConfiguration
```

Comment:
Done


On 20 May 2016 at 07:17, Andreas Håkansson <notifications@github.com> wrote:

> In samples/Nancy.Demo.Hosting.Kestrel/AppConfiguration.cs
> <https://github.com/NancyFx/Nancy/pull/2414#discussion_r63995744>:
>
> > +
> > +    public class Logging
> > +    {
> > +        public bool IncludeScopes { get; set; }
> > +        public LogLevel LogLevel { get; set; }
> > +    }
> > +
> > +    public class Smtp
> > +    {
> > +        public string Server { get; set; }
> > +        public string User { get; set; }
> > +        public string Pass { get; set; }
> > +        public string Port { get; set; }
> > +    }
> > +
> > +    public class AppConfiguration
>
> I think we should put this at the top as the filename is
> AppConfiguration.cs
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/NancyFx/Nancy/pull/2414/files/d03a128f013dd831011241475dc89419ac85db10#r63995744>
>